te_dp/src/te_sock.c

```diff
@@ -58,7 +58,7 @@ void te_create_socket_hashTbl(unsigned int size)
 
 void free_socket_node(uv_handle_t *socket_poll_handle)
 {
-    te_socket_node_t *socket_node = socket_poll_handle->data;
```

Comment:
Renaming the struct to indicate the socket_node and there by the hash is only relevant in UDP path. We could further examine and eliminate the UDP hash table also if possible.